src/dialects/websql/index.js

```diff
@@ -20,6 +20,8 @@ inherits(Client_WebSQL, Client_SQLite3);
 
 assign(Client_WebSQL.prototype, {
 
+  SqlString: require('../../query/string'),
```

Comment:
Shouldn't be necessary to set here since client base implementation already does that.